### PR TITLE
Add table of instrument additive property

### PR DESCRIPTION
**Pull request by MrAlias**
Part of #2905

Explicitly state in the metric supplementary guidelines the implied additive quality of all instrument the API provides.

### CODE REVIEW

File: specification/metrics/supplementary-guidelines.md [hunk 1]
```diff
@@ -112,9 +112,16 @@ middle. Here are some examples:
...
-additive or not. For example, [Counter](./api.md#counter) and
-[UpDownCounter](./api.md#updowncounter) are additive while [Asynchronous
-Gauge](./api.md#asynchronous-gauge) is non-additive.
+additive or not.
+
+| Instrument                                                        | Additive     |
+| ----------------------------------------------------------------- | ------------ |
+| [Counter](./api.md#counter)                                       | additive     |
+| [UpDownCounter](./api.md#updowncounter)                           | additive     |
+| [Histogram](./api.md#histogram)                                   | mixed        |
```
Comment (cijothomas) [inline specification/metrics/supplementary-guidelines.md:121]: what does mixed means?

Comment (MrAlias) [reply specification/metrics/supplementary-guidelines.md:121]: The bucket counts are additive if the buckets are the same, sum is additive, but min/max are non-additive.

Is a comment needed to explain this in doc?

Comment (MrAlias) [reply specification/metrics/supplementary-guidelines.md:121]: The term mixed is used to describe "somewhere in the middle" as stated in the sections' top paragraph. Would a different term be better?

**Review by jmacd — APPROVED** [review]

File: specification/metrics/supplementary-guidelines.md (hunk 1 above)
Comment (cijothomas) [reply specification/metrics/supplementary-guidelines.md:121]: > 

Yes this comment makes it very clear.. any chance you add it below the table?

**Review by cijothomas — APPROVED** [review]

**Pull request merged**